sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -119,13 +121,31 @@ describe.skip("Session Token", function (this: Suite) {
     // Create items using both clients so they each establish a session with the backend
     const container2 = clientB.database(dbId).container(containerId);
     await Promise.all([createItem(container), createItem(container2)]);
+  });
+
+  it("Should check if the MaxIntegratedCacheStaleness header is set.", async function () {
+    const dbId = addEntropy("maxIntegratedCacheTestDB");
+    const containerId = addEntropy("maxIntegratedCacheTestContainer");
+
+    const client = new CosmosClient({
+      endpoint,
+      key: masterKey,
+      consistencyLevel: ConsistencyLevel.Eventual,
+    });
+
+    // Create Database and Container
+    const { database } = await client.databases.createIfNotExists({
+      id: dbId,
+    });
+    const { container } = await database.containers.createIfNotExists({
+      id: containerId,
+    });
+    client.database(dbId).container(containerId);
+    const spy = sinon.spy(container, "read");
 
-    // Create an item using client
-    const id = await createItem(container);
```

Comment:
[Blocking] This removal impacts existing test right? 
May be ideal to have a separate test just for cachestaleness just to be clean and isolated.

Reply:
this is a new test

Reply:
Un-resolving it.
Sure its a new test but its still removing code from the existing test.

Reply:
Ah so you meant say revert this.

Reply:
reverted